tests/processor/tasks/test_process.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Test the process task."""
+
+from unittest.mock import patch
+
+from masu.processor.cur_process_request import CURProcessRequest
+from masu.processor.tasks.process import process_report_file
+
+from tests import MasuTestCase
+
+
+class process_report_fileTests(MasuTestCase):
```

Comment:
class name should be `camelCase`'d could be `ProcessTests` to cover all functions in process.py